app/src/main/java/com/sebastianvm/mango/util/bigdecimal/BigDecimalSerializer.kt

```diff
@@ -0,0 +1,29 @@
+package com.sebastianvm.mango.util.bigdecimal
+
+import kotlinx.serialization.KSerializer
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.descriptors.PrimitiveKind
+import kotlinx.serialization.descriptors.PrimitiveSerialDescriptor
+import kotlinx.serialization.descriptors.SerialDescriptor
+import kotlinx.serialization.encoding.Decoder
+import kotlinx.serialization.encoding.Encoder
+import java.math.BigDecimal
+
+class BigDecimalSerializer : KSerializer<BigDecimal> {
```

Comment:
BigDecimalSerializer

Reply:
Nice 💪 does this need a value converter for room?

Reply:
Not atm, since it is used only in nested in other objects for which we already have value converters.